recipes/splitting-build-compile-and-execute-code.md

```diff
@@ -0,0 +1,103 @@
+# Splitting build, compile and execute code
+
+Kysley is primarily a type-safe sql query builder.
```

Comment:
Kysley

Reply:
Nice catch! damn, first word going in! 👏